internal/clientconn/listener.go

```diff
@@ -105,17 +106,30 @@ func (l *Listener) Run(ctx context.Context) error {
 
 		// run connection
 		go func() {
+			connID := fmt.Sprintf("%s -> %s", netConn.RemoteAddr(), netConn.LocalAddr())
```

Comment:
Hmm, should `connID` be a part of the `conn` struct then?

Reply:
We don't use it otherwise, and I don't think we should. Having it as a field will be too tempting